pages/components/avatar/avatar.css

```diff
@@ -0,0 +1,62 @@
+@import url("/utils/color.css");
+.sui_avatar {
+  display: flex;
+}
+.av_txt {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+}
+.av_txt h2 {
+  color: var(--BG);
+}
+.sui_avatar {
+  border-radius: 50%;
+  overflow: hidden;
+}
+.av_square {
+  border-radius: 10px;
+}
+.av_xxl {
+  width: 14rem;
+  height: 14rem;
+}
+.av_xxl h2 {
+  font-size: 7rem;
+}
+.av_xl {
+  width: 12rem;
+  height: 12rem;
+}
+.av_xl h2 {
+  font-size: 6rem;
+}
+.av_l {
```

Comment:
this could be lg... lg is more understandable than l.

Reply:
thanks for pointing it out. I'll update it from next components.